src/js/tabs/pid_tuning.js

```diff
@@ -659,6 +684,43 @@ TABS.pid_tuning.initialize = function (callback) {
         RC_tuning.rcYawRate = parseFloat($('.pid_tuning input[name="rc_rate_yaw"]').val());
         RC_tuning.rcPitchRate = parseFloat($('.pid_tuning input[name="rc_rate_pitch"]').val());
         RC_tuning.RC_PITCH_EXPO = parseFloat($('.pid_tuning input[name="rc_pitch_expo"]').val());
+        
+        if (semver.gte(CONFIG.apiVersion, "1.43.0")) {
+            switch(self.currentRatesType) {
+                case 1: // RaceFlight
+                    RC_tuning.pitch_rate = parseFloat($('.pid_tuning input[name="pitch_rate"]').val()) / 1000;
+                    RC_tuning.roll_rate = parseFloat($('.pid_tuning input[name="roll_rate"]').val()) / 1000;
+                    RC_tuning.yaw_rate = parseFloat($('.pid_tuning input[name="yaw_rate"]').val()) / 1000;
+                    RC_tuning.rcPitchRate = parseFloat($('.pid_tuning input[name="rc_rate_pitch"]').val()) / 1000;
+                    RC_tuning.RC_RATE = parseFloat($('.pid_tuning input[name="rc_rate"]').val()) / 1000;
+                    RC_tuning.rcYawRate = parseFloat($('.pid_tuning input[name="rc_rate_yaw"]').val()) / 1000;
+                    RC_tuning.RC_PITCH_EXPO = parseFloat($('.pid_tuning input[name="rc_pitch_expo"]').val()) / 100;
+                    RC_tuning.RC_EXPO = parseFloat($('.pid_tuning input[name="rc_expo"]').val()) / 100;
+                    RC_tuning.RC_YAW_EXPO = parseFloat($('.pid_tuning input[name="rc_yaw_expo"]').val()) / 100;
+
+                    break;
+
+                case 2: // KISS
+
+                    break;
+
+                case 3: // Actual
+
+                    break;
+
```

Comment:
I'm not too sure, but if we have several options that don't do nothing, maybe is better group them at the end, and not maintain the numerical order. But this is only my opinion. For this and other similar places in the code.